src/node-lib/auth/auth0.ts

```diff
@@ -0,0 +1,12 @@
+import { initAuth0 } from "@auth0/nextjs-auth0";
+
+import getServerConfig from "@/node-lib/getServerConfig";
+
+export const auth0 = initAuth0({
+  baseURL:
+    "https://deploy-preview-2614--oak-web-application.netlify.thenational.academy/",
```

Comment:
It's hard to get the deploy preview URL without some build shenanigans. So hard coding for this demo.